courses/tasks.py

```diff
@@ -73,3 +78,21 @@ def exception_logging_generator(generator):
             log.exception("EdX API error for fetching user grades %s:", exc)
         except Exception as exp:  # pylint: disable=broad-except
             log.exception("Error fetching user grades from edX %s:", exp)
+
+
+@app.task
+def sync_courseruns_data():
+    """
+    Task to sync titles and dates for course runs from edX.
+    """
+    now = now_in_utc()
+    runs = CourseRun.objects.live().filter(
+        Q(expiration_date__isnull=True) | Q(expiration_date__gt=now)
+    )
+    success_logs, error_logs = sync_course_runs(runs)
+
+    for result_log in success_logs:
+        log.info(result_log)
+
+    for result_log in error_logs:
+        log.warning(result_log)
```

Comment:
`log.error` or `warning ` ?

Reply:
Warning is better, otherwise it will spam the Sentry logs

Reply:
Raising all errors to Sentry now.